### DISCUSSION TITLE
第一次运行nb run --reload报错

### DISCUSSION THREAD

**Discussion by Ethereal-sakura** [nonebot/discussions#360] [问答]
运行这个报错 nb run --reload
![image](https://user-images.githubusercontent.com/110333012/229278712-b4de7fff-a169-4b5c-934d-af6e7647f6cc.png)

**Comment by yanyongyu** [comment]
换一个端口试试

**Reply by Ethereal-sakura** [reply]
请问怎么切换端口，我是Windows系统

**Reply by yanyongyu** [reply]
参考文档配置：https://v2.nonebot.dev/docs/next/appendices/config#port
